lib/active_merchant/billing/gateways/xpay.rb

```diff
@@ -0,0 +1,139 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class XpayGateway < Gateway
+      self.display_name = 'XPay Gateway'
+      self.homepage_url = 'https://developer.nexi.it/en'
+
+      self.test_url = 'https://stg-ta.nexigroup.com/api/phoenix-0.0/psp/api/v1/'
+      self.live_url = 'https://xpay.nexigroup.com/api/phoenix-0.0/psp/api/v1/'
+
+      self.supported_countries = %w(AT BE CY EE FI FR DE GR IE IT LV LT LU MT PT SK SI ES BG HR DK NO PL RO RO SE CH HU)
```

Comment:
yes, there are two missing countries (Holland and Great Britain) because we don't have these country codes in our `active_merchant/lib/active_merchant/country.rb` anyway should we include them ?

Reply:
Ahh that should be because Holland is now Netherlands (NL), and Great Britain is not a country maybe you are referring to England perhaps?

Reply:
yes